### ISSUE TITLE
Usual tab should not be reused for private one

### ISSUE THREAD

**Issue by basilevs** [basilevs/reusetab-firefox#13]
### Test steps
- Open a site in a usual tab
- Pin the tab
- Open a private window
- Open a different location from the same site using an "Open link in New Private window" context menu

### Expected
Private tab is left open, address of usual tab is untouched.
### Actual
New address is opened in usual tab, private tab is closed.

https://developer.mozilla.org/en-US/docs/Mozilla/Add-ons/WebExtensions/manifest.json/incognito

**Comment by basilevs** [comment]
```
1552202542140	addons.webextension.<unknown>	ERROR	Loading extension 'null': Reading manifest: Error processing incognito: Invalid enumeration value "not_allowed" Log.jsm:679
    append resource://gre/modules/Log.jsm:679
    log resource://gre/modules/Log.jsm:360
    error resource://gre/modules/Log.jsm:368
    _logMessage resource://gre/modules/Extension.jsm:363
    logError resource://gre/modules/Extension.jsm:359
    packagingError resource://gre/modules/Extension.jsm:346
    manifestError resource://gre/modules/Extension.jsm:332
    parseManifest resource://gre/modules/Extension.jsm:601
    InterpretGeneratorResume self-hosted:1255
    next self-hosted:1210
```

**Comment by basilevs** [comment]
Recent versions of Firefox support the required manifest option.
Fixed in release 0.9.
